pages/resources/[page].js

```diff
@@ -1,75 +1,254 @@
+/* eslint-disable no-console */
 import { useEffect, useState } from 'react';
 import { useRouter } from 'next/router';
+import dynamic from 'next/dynamic';
+import { oneOfType, array, shape, string } from 'prop-types';
 import get from 'lodash/get';
-import isNumber from 'lodash/isNumber';
+import omit from 'lodash/omit';
 import Content from 'components/Content/Content';
 import Head from 'components/head';
 import HeroBanner from 'components/HeroBanner/HeroBanner';
 import Pagination from 'components/Pagination/Pagination';
-import ResourceCard from 'components/Cards/ResourceCard/ResourceCard';
-import { getResourcesPromise } from 'common/constants/api';
+import ResourceSkeletonCard from 'components/Cards/ResourceCard/ResourceSkeletonCard';
+import {
+  getResourcesPromise,
+  getResourcesByCategories,
+  getResourcesByLanguages,
+  getResourcesBySearch,
+} from 'common/constants/api';
+import { Field, Formik } from 'formik';
+import Form from 'components/Form/Form';
+import Input from 'components/Form/Input/Input';
+import Button from '../../components/Button/Button';
 import styles from '../styles/resources.module.css';
+import Select from '../../components/Form/Select/Select';
+import Alert from '../../components/Alert/Alert';
+import { RESOURCE_SEARCH } from '../../common/constants/testIDs';
 
 const pageTitle = 'Resources';
 
-function Resources() {
+const ResourceCard = dynamic(() => import('../../components/Cards/ResourceCard/ResourceCard'), {
+  loading: () => <ResourceSkeletonCard numberOfSkeletons={5} />,
+});
+
+Resources.propTypes = {
+  initialValues: shape({
+    category: string,
+    q: string,
+    languages: oneOfType([string, array]),
+    paid: string,
+  }),
+};
+
+Resources.defaultProps = {
+  initialValues: {
+    category: '',
+    q: '',
+    languages: [],
+    paid: '',
+  },
+};
+
+function Resources({ initialValues }) {
   const router = useRouter();
   const { pathname, query } = router;
-  const currentPage = parseInt(query.page, 10);
+  const { page, category, languages, paid, q } = query;
+  const currentPage = parseInt(page, 10);
 
-  if (!isNumber(currentPage)) {
-    // TODO: Handle situation where user tried some funny business, like `/resources/fuck-you`
-    throw new Error(`'currentPage' is not a number`);
+  // eslint-disable-next-line no-restricted-globals
+  if (page && isNaN(page)) {
```

Comment:
You haven't imported `isNaN` from lodash which is why it's complaining about the global. Do so, and you can remove the lint disable.